erase-install/erase-install.download.recipe

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Description</key>
+    <string>This recipe downloads for the latest version of erase-install script from its Github Releases page</string>
+    <key>Identifier</key>
+    <string>com.github.its-unibas.download.erase-install</string>
+    <key>Input</key>
+    <dict>
+        <key>NAME</key>
+        <string>erase-install</string>
+        <key>SEARCH_PATTERN</key>
+        <string>erase-install-(\d*\.\d*).pkg</string>
+        <key>GITHUB_REPO</key>
+        <string>grahampugh/erase-install</string>
+    </dict>
+    <key>MinimumVersion</key>
+    <string>0.4.2</string>
+    <key>Process</key>
+    <array>
+        <dict>
+            <key>Processor</key>
+            <string>GitHubReleasesInfoProvider</string>
+            <key>Arguments</key>
+            <dict>
+                <key>asset_regex</key>
```

Comment:
Why is the `asset_regex` needed?

Reply:
This is due to the fact, that there are several version of the erase-install script offered for each Release on the Github Releases page. Added an asset_regex makes sure that only the standard version of the script is downloaded (not the  "nopython" version or "depnotify" both of which are listed in the "Asset" section alongside the standard release).

Reply:
Could you add this into the recipe description?

Reply:
Hello, this additional info was now added to the .download recipe description.